src/main/java/our/member/service/UserAuthenticationService.java

```diff
@@ -0,0 +1,63 @@
+package our.member.service;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.server.ResponseStatusException;
+import our.member.domain.TokenEntity;
+import our.member.domain.UserEntity;
+import our.member.repository.TokenRepository;
+
+import java.time.LocalDateTime;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+public class UserAuthenticationService implements authenticationService{
+    private final TokenRepository tokenRepository;
+    private static Map<String, String> mailSender = new HashMap<>();
+
+    public UserAuthenticationService(TokenRepository tokenRepository) {
+        this.tokenRepository = tokenRepository;
+    }
+
+    @Override
+    public void sendTokenToEmail(UserEntity userEntity) {
```

Comment:
유저의 전체 정보를 가져오는 것보다, 사용자의 이메일과 사용자 이름만 받으면 사용자 엔티티를 정의하지 않고도 진행할 수 있어요!
```suggestion
    public void sendTokenToEmail(String email, String username) {
```